web/src/components/visually-hidden/visually-hidden.css

```diff
@@ -0,0 +1,13 @@
+.visually-hidden {
```

Comment:
I should add the reference to GOV.UK's code here as this is where it's stole from